src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+@use '../_variables/variables' as *;
+@use '../mixins/mixins.scss' as *;
+
+.card {
+  width: $CardWidth;
+  padding: $CardWidth * 0.01 * 16 $CardWidth * 0.01 * 8 $CardWidth * 0.01 * 8;
+  border: 1px solid $ColorCardBorder;
+
+  @include transition-options(transform);
+
+  &__image {
+    width: $CardWidth * 0.8;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: $FontMedium;
+    margin-bottom: 4px;
+
+    @include transition-options(color);
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover &__title {
+    color: $ColorDarkBlue;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $ColorSecondary;
+    margin-bottom: 16px;
+  }
+
+  &__rank {
+    display: flex;
+    justify-content: space-between;
+    align-items: stretch;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $ColorMainAccent;
+  }
+
+  &__price {
+    font-size: 12px;
+    color: $ColorSecondary;
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+
+    &-value {
+      font-size: 16px;
+      font-weight: $FontBold;
+      line-height: 18px;
+      color: $ColorMainAccent;
+    }
+  }
+
+  &__buy-button {
+    width: 100%;
+    background-color: $ColorBlueAccent;
+    color: $White;
+    text-transform: uppercase;
+
+    @include transition-options(
+      background-color,
+      outline,
+      outline-offset,
+      color,
+      cursor
+    );
+
+    &:hover {
+      background-color: $White;
```

Comment:
The variable `$White` should be written in lowercase letters with hyphens instead of camelCase, as per the code style rules.